markers/sauce.py

```diff
@@ -14,7 +17,31 @@ def pytest_addoption(parser):
 def pytest_configure(config):
     config.addinivalue_line('markers', __doc__.splitlines()[0])
     if config.option.sauce:
+        conf.env['browser']['sauce'] = True
```

Comment:
this configuration will go to "jenkins-enver" in cfme-qe-yamls as below, so that all jenkins jobs pick this when specified --sauce:
if enable_sauce:
    env['browser']['sauce'] = True